rtb/norme_di_progetto_ver0.6.0.typ

```diff
@@ -531,9 +556,177 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+
+===== Riunioni Esterne
+Le riunioni esterne coinvolgono i membri del gruppo e il proponente.\
+Per le riunioni con il proponente, viene utilizzata la piattaforma Google Meet, e l'indirizzo viene comunicato al _team_ di volta in volta. In accordo con l'azienda proponente gli incontri di fine sprint avvengono ogni due settimane, in quanto lo sprint dura due settimane, mentre a metà di ogni sprint si svolgerà un incontro di allineamento più breve per un confronto e degli aggiustamenti.
+
+===== Compiti del responsabile
+- Esporre i punti all'ordine del giorno;
+- Fare il punto su quanto fatto e sul rispetto degli obiettivi posti;
+- Pianificare le prossime attività da svolgere;
+- Assegnare i _task_ ai membri del gruppo.
+
+==== Verbali
+===== Verbali Interni
+Al fine di tracciare le discussioni svolte, le decisioni prese dal gruppo e soprattutto le motivazioni dietro ad esse, al termine di ogni incontro interno al gruppo, viene aperta una _issue_ su Github per la preparazione, la verifica e l'approvazione del verbale. Il compito di redigere il verbale, seguendo il formato indicato nella sezione 3.1 di questo documento, è affidato al responsabile, il quale deve assicurarsi di includere tutte le informazioni rilevanti discusse.
+
+===== Verbali Esterni
+Per quanto riguarda le riunioni con i proponenti, valgono le stesse regole di quelle interne, con il passaggio aggiuntivo, di cui si occupa il responsabile, di far firmare il verbale ai proponenti una volta redatto e approvato.
+
+== Miglioramento
+=== Descrizione
+Secondo lo standardG ISO/IEC 12207:1995, il processo di miglioramento nel ciclo di vita del software è finalizzato a stabilire, misurare, controllare e migliorare i processi che lo compongono. L’attività di miglioramento è composta da:
+- Analisi: Identificare le aree di miglioramento dei processi;
+- Miglioramento: Implementare le modifiche necessarie per migliorare i processi di sviluppo del software;
+
+==== Analisi dei processi
+Innanzitutto occorre stabilire una serie di processi organizzativi per l'intero ciclo di vita del software applicabili alle varie attività di progetto. Quest'ultimi devono essere documentati, come viene fatto proprio qui, e va implementato un meccanismo di controllo per sviluppare, monitorare e migliorare i processi stessi. Per garantire efficacia e continuo miglioramento dei processi il responsabile guiderà una retrospettiva di gruppo da effettuare alla fine di ogni sprint e da mettere nero su bianco per ogni sprint nel documento Piano di Progetto.
```

Comment:
- "software" da mettere in corsivo
- "sprint" da mettere in corsivo
- Non so se mettere che "guida una retrospettiva di gruppo" perché T aveva detto che era meglio venisse svolta da un singolo (chiedo conferma a @klamerja)
- Cambierei "...da mettere nero su bianco per ogni sprint...", direi tipo "...e la verbalizzerà all'interno del PdP"

Reply:
Per gli ultimi due punti verifica la nuova versione della frase

Reply:
> * [ ]  Non so se mettere che "guida una retrospettiva di gruppo" perché T aveva detto che era meglio venisse svolta da un singolo (chiedo conferma a @klamerja)

La stesura della retrospettiva è un conto. In agile, la retrospettiva viene svolta coi membri del gruppo (e anche con gli stakeholder se non sbaglio). Poi non so se sia buono dire che guida la retrospettiva, alla fine è il sinonimo di "dite tutto quello che avete da dire, problemi, note positive, siamo riusciti a finire in tempo ed etc."

Reply:
> * Cambierei "...da mettere nero su bianco per ogni sprint...", direi tipo "...e la verbalizzerà all'interno del PdP"

concordo, giusto per avere formalità

Reply:
Perfetto, risolto